spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/DataTypeConverter.scala

```diff
@@ -0,0 +1,58 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector
+import com.datastax.spark.connector.cql.ColumnDef
+import com.datastax.spark.connector.types.FieldDef
+import org.apache.spark.Logging
+import org.apache.spark.sql.types.StructField
+import org.apache.spark.sql.{types => catalystTypes}
+
+/** Convert Cassandra data type to Catalyst data type */
+object DataTypeConverter extends Logging {
```

Comment:
Maybe put it into ColumnType object so it is together with all the code for type conversions?